x-pack/plugins/search_playground/server/lib/get_chat_params.ts

```diff
@@ -62,31 +70,57 @@ export const getChatParams = async (
         context: true,
         type: 'openai',
       });
+      questionRewritePrompt = QuestionRewritePrompt({
+        type: 'openai',
+      });
       break;
     case BEDROCK_CONNECTOR_ID:
-      const llmType = 'bedrock';
-      chatModel = new ActionsClientLlm({
+      llmType = 'bedrock';
+      chatModel = new ActionsClientSimpleChatModel({
         actionsClient,
         logger,
         connectorId,
         model,
-        traceId: uuidv4(),
         llmType,
         temperature: getDefaultArguments(llmType).temperature,
+        streaming: true,
       });
       chatPrompt = Prompt(prompt, {
         citations,
         context: true,
         type: 'anthropic',
       });
+      questionRewritePrompt = QuestionRewritePrompt({
+        type: 'anthropic',
+      });
+      break;
+    case GEMINI_CONNECTOR_ID:
+      llmType = 'gemini';
+      chatModel = new ActionsClientSimpleChatModel({
+        actionsClient,
+        logger,
+        connectorId,
+        model,
+        llmType,
+        temperature: getDefaultArguments(llmType).temperature,
+        streaming: true,
+      });
+      chatPrompt = Prompt(prompt, {
+        citations,
+        context: true,
+        type: 'gemini',
```

Comment:
can you use llmType here?

Reply:
personally i think its better here being explicit.